runtests.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 """
 runtests.py [OPTIONS] [-- ARGS]
-
+Note: MSVC compiler versions needed on Windows for runtests.py to execute successfully
```

Comment:
How about just
```
Note: A C compiler is needed when running tests that require a build.
```
Note that it is possible the test the installed NumPy with the `--no-build` option.